cmd/env_test.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_validateShellName(t *testing.T) {
+	tests := []struct {
+		name       string
+		str        string
+		shouldFail bool
+	}{
+		{name: "strings with spaces should fail", str: "invalid strings", shouldFail: true},
+		{name: "strings with only underscores should pass", str: "valid_string", shouldFail: false},
+		{name: "strings with dashes should fail", str: "validish-string", shouldFail: true},
+		{name: "strings that start with numbers should fail", str: "1invalidstring", shouldFail: true},
```

Comment:
do we need a strings that include a number (but don't start with) should pass test?

Reply:
eh, feels redundant; it's a valid character -- what matters is it **doesn't start** with a number so that's (I believe) the only case we actually need to screen for.